plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherPlugin.kt

```diff
@@ -28,16 +29,19 @@ import groovy.lang.GroovyObject
 import org.gradle.api.Plugin
 import org.gradle.api.Project
 import org.gradle.api.plugins.ExtensionAware
-import org.gradle.kotlin.dsl.the
+import org.gradle.kotlin.dsl.withType
 
 @Suppress("unused") // Used by Gradle
 class PlayPublisherPlugin : Plugin<Project> {
     override fun apply(project: Project) {
         validateRuntime()
 
-        val android = requireNotNull(project.the<AppExtension>()) {
```

Comment:
Note: when using `plugins {}` DSL `AppExtension` is not immediately available because `com.android.application` is not applied yet.